scripts/west_commands/flash.py

```diff
@@ -28,6 +29,5 @@ def do_add_parser(self, parser_adder):
 
     def do_run(self, my_args, runner_args):
         build_dir = get_build_dir(my_args)
-        domains = load_domains(build_dir).get_domains(my_args.domain,
-                                                      default_flash_order=True)
-        do_run_common(self, my_args, runner_args, domains=domains)
+        domains_file = Path(build_dir) / 'domains.yaml'
+        do_run_common(self, my_args, runner_args, domain_file=domains_file)
```

Comment:
`domains_file` basically acts as a bool, so there's no point in it being a `Path` object.

Reply:
Updated

Reply:
Unresolved